chapters/en/slides/module8/module8_03.md

```diff
@@ -329,36 +307,34 @@ Equation {eq}`eq_label` is an example of a named equation.
 ~~~
 
 Notes:
-
 If you have created an equation with a label, you can link to it from within your text and across pages.
 
 You can refer to the equation using the label that you’ve provided by using `` {eq}`eq_label` ``, just like ``{numref}`jupiter-figure` `` that we used with images.
 
 <br>
 
-Here is how we add equations to `mymarkdownfile.md`:
+You can see in the slide how we add equations to `my-markdown-file.md`:
 
 ---
 
-Once you add these equations to `mymarkdownfile.md` and rebuild your Jupyter Book, your book will look like:
+## Cross-referencing equations
+
+Once you add these equations to `my-markdown-file.md` and rebuild your Jupyter Book, your book will look like:
 
 <br>
 
 <img style="border:1px solid black;" src="/module8/equation.png" width="750"></img>
 
 Notes:
 
-Once you add these equations to `mymarkdownfile.md` and rebuild your Jupyter Book, your book will look like:
-
-<img style="border:1px solid black;" src="/module8/equation.png" width="750"></img>
+Once you add these equations to `my-markdown-file.md` and rebuild your Jupyter Book, your book will look like the screenshot in this slide.
+Note the equation numbering pointed at by the red arrow.
 
 ---
 
 ## Margin content
 
-Sometimes it is helpful to add notes to the margin of a page. This is similar to _footnotes_ in a conventional book.
-
-In Jupyter Book, you can do that using `{margin}` blocks in a markdown file:
+Margin content is similar to _footnotes_ in a conventional book:
 
```

Comment:
I think this slide deck is quite long at 25 slides. What to you think about splitting it somewhere around here? It is maybe a bit arbitrary where to split and not technically necessary, but I think it will facilitate both recording and learning to have two shorter decks.

Reply:
I split this slide deck into two parts, one part addressing only the essentials, and one part for more advanced features (from the "Margin content" slide onwards), and added one additional MCQ for the 2nd part.